lib/oli/delivery/sections/certificates/workers/check_certification.ex

```diff
@@ -5,11 +5,30 @@ defmodule Oli.Delivery.Sections.Certificates.Workers.CheckCertification do
     max_attempts: 1
 
   alias Oli.Delivery.GrantedCertificates
+  alias Oli.Delivery.Sections.Certificates.Workers.CheckCertification
 
   @impl Oban.Worker
   def perform(%Oban.Job{args: %{"user_id" => user_id, "section_id" => section_id}}) do
     GrantedCertificates.has_qualified(user_id, section_id)
 
     :ok
   end
+
+  def restart_certificate_check(user_id, section_id) do
+    Ecto.Adapters.SQL.query!(
```

Comment:
Should we wrap this in a database transaction to ensure we guard against the (unlikely) case where a job is inserted between the check and the insert?